log/handler_glog.go

```diff
@@ -207,21 +207,21 @@ func (h *GlogHandler) Log(r *Record) error {
 	}
 	// Check callsite cache for previously calculated log levels
 	h.lock.RLock()
-	lvl, ok := h.siteCache[r.Call.PC()]
+	lvl, ok := h.siteCache[r.Call.Frame().PC]
```

Comment:
Could you paste what the original errors were here?

Reply:
```
log/handler_glog.go:210:25: r.Call.PC is deprecated: Use Call.Frame instead.  (SA1019)
log/handler_glog.go:218:17: r.Call.PC is deprecated: Use Call.Frame instead.  (SA1019)
log/handler_glog.go:224:16: r.Call.PC is deprecated: Use Call.Frame instead.  (SA1019)
log/logger.go:86:30: error strings should not be capitalized (ST1005)
```